pkg/tfvars/libvirt/libvirt.go

```diff
@@ -50,6 +52,12 @@ func TFVars(masterConfig *v1beta1.LibvirtMachineProviderConfig, osImage string,
 		MasterVcpu:   strconv.Itoa(masterConfig.DomainVcpu),
 	}
 
+	// Power PC systems typically require more memory because the page size is 64K and not the default 4K
+	// TODO: FIXME: need to make ppc64le a supported architecture - need to do this when adding support for multi-arch images. Until then use this
```

Comment:
please don't keep FIXME in code, rather create a github issue, or JIRA issue that tracks details about the problem and include the link here.
external tracker allows us to comeback and fix this when that issue is fixed.

Reply:
yes.this was more a reminder for me to get the multi-arch support in.Opened a BZ for this and referenced it.